platform/android/java/editor/build.gradle

```diff
@@ -0,0 +1,53 @@
+// Gradle build config for Godot Engine's Android port.
+apply plugin: 'com.android.application'
+
+dependencies {
+    implementation libraries.kotlinStdLib
+    implementation libraries.androidxFragment
+    implementation project(":lib")
+}
+
+android {
+    compileSdkVersion versions.compileSdk
+    buildToolsVersion versions.buildTools
+    ndkVersion versions.ndkVersion
+
+    defaultConfig {
+        // The 'applicationId' suffix allows to install Godot 3.x(x3) and 4.x(x4) on the same device
+        applicationId "org.godotengine.editor.x3"
+        minSdkVersion versions.minSdk
+        //noinspection ExpiredTargetSdkVersion - Restrict to version 29 until https://github.com/godotengine/godot/pull/51815 is submitted
+        targetSdkVersion 29 // versions.targetSdk
```

Comment:
I guess that means this could be published on Google Play until this is solved? (Not that it's a problem right now, we don't have explicit plans to do so at this time.)

Reply:
Yes, that's correct. The current plan is to stick to target sdk 29 until we're done with alpha and beta releases for the editor.